### PR TITLE
Typescript example (compared with master)

### PR DESCRIPTION
**Pull request by abeaudoin2013**
Hey Alex,
Here it is.

Thanks for creating this awesome library! Can't wait for the docs to come out. 

Andy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1409,6 +1409,8 @@ import type { DroppableProvided } from 'react-beautiful-dnd';
 
 If you are using [TypeScript](https://www.typescriptlang.org/) you can use the community maintained [DefinitelyTyped type definitions](https://www.npmjs.com/package/@types/react-beautiful-dnd). [Installation instructions](http://definitelytyped.org/).
 
+Here is an [example written in typescript](https://github.com/abeaudoin2013/react-beautiful-dnd-multi-list-typescript-example).
```
Comment (alexreardon) [inline README.md]: *Here is

**Comment by alexreardon** [comment]
So close 👍

File: README.md (hunk 1 above)
Comment (alexreardon) [reply README.md]: It looks like it is just an issue with the diff view

Comment (alexreardon) [reply README.md]: The rich diff is correct:
https://github.com/atlassian/react-beautiful-dnd/pull/666/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8

**Comment by abeaudoin2013** [comment]
👍

**Pull request merged**